### PR TITLE
Dc description may be an abstract

### PR DESCRIPTION
**Pull request by janbj**

### CODE REVIEW

File: metadata-transformer/src/test/java/no/unit/nva/metadata/service/MetadataServiceTest.java [hunk 1]
```diff
@@ -40,23 +41,12 @@ public void getMetadataJsonReturnsCreatePublicationRequest()
         URI uri = prepareWebServerAndReturnUriToMetdata(TEST_REVIEW_PAPER_HTML);
         Optional<CreatePublicationRequest> request = metadataService.getCreatePublicationRequest(uri);
 
-        assertKnownMetadataMappingsFromReviewPaperHtml(request.get());
+        File file = new File(TEST_REVIEW_PAPER_JSON);
```
Comment (axthosarouris) [inline metadata-transformer/src/test/java/no/unit/nva/metadata/service/MetadataServiceTest.java:44]: After the fix we made in commons, the following should work
```
String json = IoUtils.stringFromResources(Path.of(TEST_REVIEW_PAPER_JSON));
```
if not you can use
```
//imported  IoUtils.*
String json = streamToString(inputStreamFromResources(TEST_REVIEW_PAPER_JSON));
```

File: metadata-transformer/src/test/java/no/unit/nva/metadata/service/MetadataServiceTest.java [hunk 2]
```diff
@@ -66,19 +56,12 @@ public void getMetadataJsonFromUpperCaseTagMetadataReturnsCreatePublicationReque
         URI uri = prepareWebServerAndReturnUriToMetdata(UPPER_CASE_DC_HTML);
         Optional<CreatePublicationRequest> request = metadataService.getCreatePublicationRequest(uri);
 
-        assertKnownMetadataMappingsFromUpperCaseDcHtml(request.get());
+        File file = new File(UPPER_CASE_DC_JSON);
```
Comment (axthosarouris) [inline metadata-transformer/src/test/java/no/unit/nva/metadata/service/MetadataServiceTest.java:59]: ditto

File: metadata-transformer/src/test/java/no/unit/nva/metadata/service/MetadataServiceTest.java (hunk 1 above)
Comment (axthosarouris) [reply metadata-transformer/src/test/java/no/unit/nva/metadata/service/MetadataServiceTest.java:44]: Based on the title of the PR, it seems that one of these tests should be parametrized for the different variants. This is just a gut feeling though.

**Review by axthosarouris — CHANGES_REQUESTED** [review]

File: metadata-transformer/src/test/java/no/unit/nva/metadata/service/MetadataServiceTest.java (hunk 1 above)
Comment (brinxmat) [reply metadata-transformer/src/test/java/no/unit/nva/metadata/service/MetadataServiceTest.java:44]: Agreed, there is a necessity to test a broad range of resources with a number of expected outcomes. I'm not sure that flat files is the optimal way for targeted testing, though.

**Review by brinxmat — APPROVED** [review]

**Review by axthosarouris — APPROVED** [review]

**Review by brinxmat — APPROVED** [review]

**Pull request merged**